static/js/settings.js

```diff
@@ -1,6 +1,6 @@
 "use strict";
 
-const moment = require("moment-timezone");
+const {listTimeZones} = require("timezone-support");
```

Comment:
`timezone-support` has a lot of data we don’t need and adds some 900 KB to the bundle. It compresses well, but that still takes memory and parsing time. Maybe we should generate the list on the server side (like `pygments_data`).

Reply:
Oh , Should I generate the timezones value in a <code>json</code> file and then process it here ? (as <code>pygments_data</code> as created in <code>./static/generated/pygments_data.json</code> )